code/datums/riding.dm

```diff
@@ -100,10 +118,14 @@
 //atv
 /datum/riding/atv
 	keytype = /obj/item/key
-	generic_pixel_x = 0
-	generic_pixel_y = 4
 	vehicle_move_delay = 1
 
+/datum/riding/atv/get_offsets(pass_index) // list(dir = x, y, layer)
+	switch(pass_index)
+		if(1)
+			return list("[NORTH]" = list(0, 4), "[SOUTH]" = list(0, 4), "[EAST]" = list(0, 4), "[WEST]" = list( 0, 4))
```

Comment:
actually i kept the switches because if someone wanted to add more passengers it's here for u 

i like it let's keep it please